src/EditorFeatures/CSharpTest/CodeActions/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -1085,6 +1085,43 @@ private void Init()
             }.RunAsync();
         }
 
+        [Fact]
+        [WorkItem("https://github.com/dotnet/roslyn/issues/71636")]
+        public async Task TestUpdateExpressionBody4()
+        {
+            await VerifyCS.VerifyRefactoringAsync(
+                """
+                using System;
+
+                class C
+                {
+                    int M(int v, [||]string s) => v switch
```

Comment:
can you add a test with a conditional expression as well, of the forms:

```
 int M(bool v, [||]string s) => v ? 0 : 1;
 int M(bool v, [||]string s) => v ?
    0 :
    1;
 int M(bool v, [||]string s) => v
    ? 0
    : 1;
```

And variations where the expr starts on the next line.  And variations where the `=>` is on the next line indented?

Reply:
Done.